explainer.md

```diff
@@ -116,10 +116,15 @@ const xrGlBinding = new XRWebGLBinding(xrSession, gl);
 const layer = xrGlBinding.createProjectionLayer({ textureType: "texture-array" });
 ```
 
-Layer types other than an `XRProjectionLayer` must be given an explicit pixel width and height, as well as whether or not the image should be stereo or mono. This is because those properties cannot be inferred from the hardware or layer type as they can with an `XRProjectionLayer`.
+Layer types other than an `XRProjectionLayer` must be given an explicit pixel width and height, as well as whether or not the image should be stereo or mono. This is because those properties cannot be inferred from the hardware or layer type as they can with an `XRProjectionLayer`. Additionally, the layer's space needs to be defined as it should be anchored to the world and not the viewer's gaze by default.
```

Comment:
The other changes are fine but this is sentence is not correct as not every space anchors content to the world. They can also anchor it to the viewer (headlock), a controller or a joint for instance.

Reply:
Updated the sentence to not make any specific assumptions about where the the content is anchored.